includes/platforms/configuration/options/allow-urls.mdx

```diff
@@ -1 +1,15 @@
-A list of strings or regex patterns that match error URLs which should exclusively be sent to Sentry. If you use this option, only errors whose entire file URL contains (string) or matches (regex) at least one entry in the list will be sent. As a result, if you add `'foo.com'` to it, it will also match on `https://bar.com/myfile/foo.com`. Keep in mind that this only applies for captured exceptions, not raw message events. By default, all errors are sent.
+An array of strings or regex patterns that match the URLs of scripts where errors have been created.
+Only errors that have been created on these URLs will be sent sent to Sentry.
+If you use this option, only Errors whose top stack frame file URL contains (string) or matches (regex) at least one entry in the array will be sent.
+
```

Comment:
```suggestion
An array of strings or regex patterns that match the URLs of scripts where errors have been created.
Only errors that have been created on these URLs will be sent to Sentry.
If you use this option, at least one entry in the array will be sent if an error's top stack frame file URL contains (string) or matches (regex).

```

Reply:
I think I know what you are getting at but the suggestion makes the content logically wrong. I took another stab: 9275103e631d1f295b24fc4dc2579d5f4abb2800